test/common/upstream/health_checker_impl_test.cc

```diff
@@ -2439,7 +2885,7 @@ TEST(TcpHealthCheckMatcher, match) {
   EXPECT_TRUE(TcpHealthCheckMatcher::match(segments, buffer));
 }
 
-class TcpHealthCheckerImplTest : public testing::Test, public HealthCheckerTestBase {
+class TcpHealthCheckerImplTest : public testing::Test, public TcpHealthCheckerImplTestBase {
```

Comment:
are the mock timers in this class needed in the base? if you're not sure yet -- maybe scope this down to HTTP and leave this for the TCP PR?

Reply:
Yup - let me scope it down to http.